test/issues/issues-test.js

```diff
@@ -508,4 +508,12 @@ describe("issues", function () {
             sinon.assert.calledWith(spy, secondObj);
         });
     });
+
+    describe("#1796 - cannot stub Array.prototype.sort", function () {
+        it("it should not fail with RangeError", function () {
+            var stub = sinon.stub(Array.prototype, "sort");
```

Comment:
This stub needs to `.callThrough()` to trigger the `RangeError`, right?

Reply:
Nope, `.callThrough()` is not necessary to trigger the `RangeError`.